inc/functions.php

```diff
@@ -549,6 +549,64 @@ function my_date($format, $stamp=0, $offset="", $ty=1, $adodb=false)
 	return $date;
 }
 
+/**
+ * Get a mail handler instance, a MyBB's built-in SMTP / PHP mail hander or one created by a plugin.
+ * @param bool $use_buitlin Whether to use MyBB's built-in mail handler.
+ *
+ * @return object A MyBB's built-in mail handler or one created by plugin(s).
+ */
+function &get_mybb_mailhandler($use_buitlin = false)
```

Comment:
I think the `mybb_` prefix within this function is superfluous, specially in the function and hook names.

Reply:
Oh, yes! Using `my_` as prefix would be better which also consists with the current function naming convention.